org/mozilla/jss/ssl/SSLProtocolVariant.java

```diff
@@ -16,5 +16,5 @@
 
     private SSLProtocolVariant(int val) { _enum = val; }
 
-    int getEnum() { return _enum; }
+    public int getEnum() { return _enum; }
```

Comment:
If we want to mimic Java Enum style, we probably can add a `value()` that returns `_enum`.

Reply:
Yeah, a future improvement would be to make these actual enums, but for the time being, I think we'll leave it as-is.